### PR TITLE
Fix language labels

### PR DESCRIPTION
**Pull request by chinwenkai**
Having difficulties implementing translation for the chart pop up labels. Amended to show numbers/values only.
Increased screen width coverage for graphs.
image

### CODE REVIEW

File: pong/media/profile_pictures/redpanda.jpg
Comment (barbayjuliette) [inline pong/media/profile_pictures/redpanda.jpg:1]: You removed this picture. You need to put it back!

**Comment by barbayjuliette** [comment]
Hover on the pie graphs is not accurate. 
I hover the blue, it the hover is for the red.

Game Margin Distribution of all games:
Add a description of what it shows. Because it is not so clear what it is trying to show.

Size of the graphs on the VM? Did you fix it and test it in the VM? 
I cannot test it now

**Review by barbayjuliette — CHANGES_REQUESTED** [review]
See comments

**Comment by barbayjuliette** [comment]
Remove all your console.log

**Comment by chinwenkai** [comment]
The charts are larger on the VM after changing from
<div class="col-12 col-md-6"> to <div class="col-12 col-md-8">
image

This seems to cause pop up misalignment on laptop, but no alignment issues on 42 VM.

**Comment by chinwenkai** [comment]
Hover accuracy fixed

Game Margin Distribution description amended.

size tested on VM

**Comment by chinwenkai** [comment]
console.log removed

**Comment by barbayjuliette** [comment]
> Issue encountered on 42 VM?

In 42 VM for yellow part of pie, in Firefox

**Comment by barbayjuliette** [comment]
Okay, now looks okay in VM

**Comment by barbayjuliette** [comment]
Just need to change the style to Bootstrap, then we can merge

**Comment by barbayjuliette** [comment]
@chinwenkai 
The pie graphs do not have the same size in Firefox and Chrome.
Too big in Firefox. Make it same size in both.

**Comment by barbayjuliette** [comment]
The hover also doesn't work properly in Firefox. 
Especially on the Game margin graph, yellow color

**Comment by chinwenkai** [comment]
https://github.com/user-attachments/assets/708be724-1add-4124-90bd-e3b4ce4c92eb

Firefox pop up seems ok for my side

**Comment by chinwenkai** [comment]
Issue encountered on 42 VM?

**Comment by chinwenkai** [comment]
styles changed to bootstrap class in dashboard.html

**Comment by barbayjuliette** [comment]
@chinwenkai remove from styles

**Comment by chinwenkai** [comment]
Removed from chart widths from styles.css

**Comment by barbayjuliette** [comment]
Screenshot 2024-11-03 at 22 58 00
Screenshot 2024-11-03 at 22 56 25
Labels missing

**Comment by barbayjuliette** [comment]
<hr class="flex-shrink-0 border-0" style="border: 10px solid rgba(54, 162, 235, 0.6); width: 2%; margin: 0; opacity: 1;">

- You have border twice
- It is not good to use inline styles, some can be replaced with bootstrap (border: 10px solid margin opacity)

**Comment by barbayjuliette** [comment]
To do this in Bootstrap:

   width: 100%;
    height: auto;
    
    You just need to add 2 classes, you do not need to redo everything!
    Jut a google search with bootstrap will give you the answer...

**Review by barbayjuliette — APPROVED** [review]
ok

**Pull request merged**